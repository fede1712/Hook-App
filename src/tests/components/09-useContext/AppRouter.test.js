import React from 'react'
import { mount } from 'enzyme'
import { AppRouter } from '../../../components/09-useContext/AppRouter'
import { user } from '../../fixtures/demoTodos'
import { UserContext } from '../../../components/09-useContext/UserContext'


describe('Tests in <AppRouter />', () => {

    const wrapper = mount(
        <UserContext.Provider value={{ user }}>
            <AppRouter />
        </UserContext.Provider>
    )

    test('should render <AppRouter /> correctly', () => {

        expect(wrapper).toMatchSnapshot()

    })


})
