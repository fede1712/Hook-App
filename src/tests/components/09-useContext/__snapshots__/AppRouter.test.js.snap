// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <AppRouter /> should render <AppRouter /> correctly 1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <div
      className="container-fluid"
    >
      <div
        className="collapse navbar-collapse"
        id="navbarNav"
      >
        <ul
          className="navbar-nav"
        >
          <li
            className="nav-item"
          >
            <a
              aria-current="page"
              className="nav-link active"
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/about"
              onClick={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/login"
              onClick={[Function]}
            >
              Login
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>,
  <div>
    <h1>
      Home Screen
    </h1>
    <hr />
    <pre>
      {
   "name": "fede",
   "email": "fede@fede.com"
}
    </pre>
  </div>,
]
`;
